test/test-call-forwarding

```diff
@@ -12,7 +13,9 @@ def property_changed(property, value):
 def print_properties(cf):
 	properties = cf.GetProperties()
 
-	for p in properties:
+	for p in ["VoiceBusy", "VoiceNoReplyTimeout", "VoiceNotReachable",
+			"VoiceUnconditional", "VoiceNoReply",
+			"ForwardingFlagOnSim"]:
```

Comment:
Why did you do this change? The set you use between the brackets is exactly the same contained in "properties".